zstacklib/zstacklib/utils/ovn.py

```
@@ -299,3 +306,214 @@ def addUplink(self, portPciMap, bondMode, lacpmode, ip, netmask, brName="br-phy"
+    ret = None
+    try:
+        with open(path, 'r') as f:
+            ret = f.read().rstrip()
+    except Exception as e:
+        logger.warn(str(e))
+        if not supressRaise:
+            raise OvsError(str(e))
+
+    return ret
+
+class OvsDpdkEnv(object):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)